tests/forgotPassword_test.js

```diff
@@ -11,6 +11,11 @@ Don't forget to create also Page Object structure in forgotPasswordPage.js and c
 
 Feature('Forgot Password');
 
-xScenario('test something', ({ I }) => {
-
+Scenario.only('Forgoten password', ({ I , ForgotPasswordPage}) => {
```

Comment:
Remove Only :-D

Reply:
http://www.reactiongifs.com/wp-content/uploads/2013/11/thank-you-for-your-input.gif